### PR TITLE
📝 Scribe: Add tests for email verification

### PR DESCRIPTION
**Pull request by GarethClarridge**
Identified that the email verification process in `AuthenticatedSessionController@verifyEmail` was under-tested. Added `tests/Feature/Auth/EmailVerificationTest.php` covering the happy path and various failure/edge cases. Verified that all tests pass, including the full suite, and that code quality tools (PHPStan, Pint) are clean.

---
*PR created automatically by Jules for task [13425380653143616226](https://jules.google.com/task/13425380653143616226) started by @GarethClarridge*

### CODE REVIEW

File: tests/Feature/Auth/EmailVerificationTest.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        $user = User::factory()->create(['email_verified_at' => null]);
+
+        $verificationUrl = URL::signedRoute('verification.verify', [
+            'id' => $user->id,
+            'hash' => 'invalid-hash',
+        ]);
+
+        $this->actingAs($user)->get($verificationUrl);
+
+        $this->assertFalse($user->fresh()->hasVerifiedEmail());
```
[the remark this answers is not in the document]
Comment (GarethClarridge) [reply tests/Feature/Auth/EmailVerificationTest.php]: @jules please fix

File: tests/Feature/Auth/EmailVerificationTest.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    #[Test]
+    public function email_is_not_verified_with_invalid_signature(): void
+    {
+        $user = User::factory()->create(['email_verified_at' => null]);
+
+        $verificationUrl = URL::route('verification.verify', [
+            'id' => $user->id,
+            'hash' => sha1($user->getEmailForVerification()),
+        ]).'&signature=invalid';
```
[the remark this answers is not in the document]
Comment (GarethClarridge) [reply tests/Feature/Auth/EmailVerificationTest.php]: @jules please fix
